libbpf-rs/src/object.rs

```diff
@@ -89,45 +145,54 @@ impl Map {
     /// Same as [`Map::lookup()`] except this also deletes the key from the map.
     ///
     /// `key` must have exactly [`Map::key_size()`] elements.
-    pub fn lookup_and_delete(&mut self, _key: &[u8], _opts: MapOptions) -> Result<Option<Vec<u8>>> {
+    pub fn lookup_and_delete(&mut self, _key: &[u8], _flags: MapFlags) -> Result<Option<Vec<u8>>> {
```

Comment:
btw, lookup can take BPF_F_LOCK flag, so I think lookup() needs to take flags as well

Reply:
ok